apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -40,13 +40,12 @@ const DATE_PARSING_FORMATS = [
 const READINGS_LIMIT = 500
 
 const ERRORS = {
-    TOO_MUCH_READINGS: (sentCount) => ({
+    TOO_MUCH_READINGS: {
         code: BAD_USER_INPUT,
         type: TOO_MUCH_READINGS,
         message: `Too much readings. Maximum is ${READINGS_LIMIT}.`,
         messageForUser: 'api.meter.registerMetersReadings.TOO_MUCH_READINGS',
```

Comment:
```suggestion
        messageForUser: 'api.meter.registerMetersReadings.TOO_MANY_READINGS',
```

Reply:
too much to do it in all places in this pr